types/src/key.rs

```diff
@@ -700,6 +746,7 @@ mod serde_helpers {
                 BinaryDeserHelper::Bid(account_hash) => Key::Bid(account_hash),
                 BinaryDeserHelper::Withdraw(account_hash) => Key::Withdraw(account_hash),
                 BinaryDeserHelper::Dictionary(addr) => Key::Dictionary(addr),
+                BinaryDeserHelper::SystemContractRegistry => Key::SystemContractRegistry,
             }
         }
     }
```

Comment:
The following need to be updated to include coverage for the new key variant, and also please update them to cover the `Dictionary` variant as that seems to have been overlooked:
* `KEYS`
* `should_display_key()`
* `should_fail_to_parse_key_from_str()`
* `key_to_json()`
* `serialization_roundtrip_json()`